src/enc_png.c

```diff
@@ -85,27 +100,35 @@ int SUB_NAME(char *data,g2int *width,g2int *height,g2int *nbits,char *pngbuf)
        return (-2);
     }
 
-/*     Set Error callback   */
+/*
```

Comment:
You shouldn't have to do anything to the comments in the code itself when doxygenating the file. They can just be left alone.